core/src/main/scala/ai/h2o/sparkling/backend/api/DELETERequestBase.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ai.h2o.sparkling.backend.api
+
+import ai.h2o.sparkling.utils.ScalaUtils.withResource
+import com.google.gson.Gson
+import javax.servlet.http.{HttpServletRequest, HttpServletResponse}
+
+private[api] trait DELETERequestBase extends ServletBase {
+
+  def handleDeleteRequest(request: HttpServletRequest): Any
+
+  override def doDelete(req: HttpServletRequest, resp: HttpServletResponse): Unit = {
```

Comment:
The body of `doDelete` and [`doGet`](https://github.com/h2oai/sparkling-water/pull/2093/files#diff-ff17f4222dc68216fd015c4db186a978R27) seems to be identical.